### ISSUE TITLE
Create Basic Room Scene

### ISSUE THREAD

**Issue by Liberontissauri** [Liberontissauri/PiuPiuProject#6]
This should be a scene where there is a room for testing. It should already have the correct sprites for walls and floor of a normal castle.
This room should just be a large empty space, with maybe some lights and shadows if you feel like implementing them, if not they can also be done later on.

Requirements:
- Room with collisions on walls
- Space for spawning enemies
    - Enemies and enemy spawning should not be implemented on this issue
- Player Spawn Management
    - Basically have a way to change the position where the player spawns in the room
